lms/static/sass/_header.scss

```diff
@@ -0,0 +1,365 @@
+            }
+
+            a:focus{
+              outline: none;
+              color: theme-color("primary");
+            }
+
+            &:not(:last-child) {
+              border-bottom: 1px solid theme-color("light");
+            }
+          }
+        }
+      }
+    }
+    .hamburger-menu {
+      display: none;
+    }
+  }
+
+  // Responsive styling for mobile
+  @include media-breakpoint-down(sm) {
+    // Styles for the transition from open to collapsed
+    .nav-links {
+      display: none;
+      -webkit-transition: height 2.0s;
+      -moz-transition: height 2.0s;
+      transition: height 2.0s;
+
+      &.nav-links-expanded {
+        display: block;
+      }
+
+      .nav-item-not-collapsible { // Hide any items that we do not want to collapse
+        display: none;
+      }
+
+      .nav-item-not-styled { // Remove specific styles from any items that we do not want to style on mobile
+        margin-top: 0;
+      }
+    }
+
+    // Display and style all mobie menu elements
+    .nav-item,
+    .dropdown-nav-item {
+      display: inline-block;
+      width: 100%;
+      border: none;
+      padding: $baseline/5 0 0;
+      margin: $baseline*0.75 $baseline 0;
+      color: theme-color("primary");
+      border: none;
+      text-decoration: none;
+      background-color: transparent;
+      font-weight: normal;
+      font-size: 20px;
+
+      a {
+        text-decoration: none;
+
+        .active, // Indicates if we are currently on one of the pages
+        &:hover{
+          color: theme-color("dark");
+        }
+      }
+
+      &:hover {
+        color: theme-color("dark");
+        font-weight: 600;
+      }
+    }
+
+    // Custom focus behavior for tabbing through elements (links, dropdown links, and buttons)
+    .nav-item a:focus,
+    .dropdown-nav-item a:focus,
+    a.nav-item:focus {
+      outline: none;
+      color: theme-color("dark");
+      font-weight: 600;
+    }
+
+
+    // Custom styling to ensure dropdown elements match other nav elements in collapsed mode
+    .dropdown-user-menu {
+      display:inline !important;
+
+      .dropdown-nav-item {
+        @include margin-left(0);
+      }
+
+      a:first-child{
+        .dropdown-nav-item {
+          margin-top: 0;
+        }
+      }
+    }
+
+    // Display the menu icon and allow for transition to an X on click
+    .hamburger-menu {
+      @include right(22px);
+      position: absolute;
+      top: 24px;
+      width: 30px;
+      height: 20px;
+      -webkit-transform: rotate(0deg);
+      -moz-transform: rotate(0deg);
+      -o-transform: rotate(0deg);
+      transform: rotate(0deg);
+      -webkit-transition: .5s ease-in-out;
+      -moz-transition: .5s ease-in-out;
+      -o-transition: .5s ease-in-out;
+      transition: .5s ease-in-out;
+      cursor: pointer;
+      outline: none;
+
+      &:focus,
+      &:hover {
+        span {
+          background: theme-color("dark");
+          height: 3px;
+        }
+      }
+
+      span {
```

Comment:
I'd target this to a class as there could be multiple spans (in the future, if not now).